src/Surfnet/StepupGateway/GatewayBundle/Tests/TestCase/GatewaySamlTestCase.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Surfnet\StepupGateway\GatewayBundle\Tests\TestCase;
+
+use Mockery;
+use PHPUnit\Framework\TestCase;
+use SAML2\Compat\ContainerSingleton;
+use SAML2\Configuration\PrivateKey;
+use Surfnet\StepupGateway\GatewayBundle\Tests\Mock\Saml2ContainerMock;
+use Symfony\Component\HttpKernel\Tests\Logger;
+
+class GatewaySamlTestCase extends TestCase
```

Comment:
A valuable addition, this base class for authentication related tests! Some thoughts:
- Please add the copyright statement
- Remove empty doc blocks or write them :nerd_face: 
- You are missing a whitespace at the bottom of the file.

Reply:
> I haven't found copyright statement in tests so far?

That should not be the case.

See: https://github.com/OpenConext/Stepup-Deploy/wiki/License-And-Copyright